src/lib.rs

```diff
@@ -310,6 +326,20 @@ impl Mold {
       }
     }
 
+    // TODO not sure if includes should be updated here,
+    // since they're always automatically cloned (which
+    // means they're updated?)
```

Comment:
This method is to fetch/checkout stuff after the upstream repo has been updated. eg:

```
$ mold --clone # clones all includes / groups
# upstream include / groups are updated
$ mold # existing includes / groups are *not* updated
$ mold --update # fetch / checkout to update
```

Reply:
I do wonder if there's maybe some way to simplify this, though. Right now, `--clone` will only clone things that *don't exist* yet, while `--update` will only update things that *do exist*.